Pa3/Durchführung/Project/src/app/State/DisplayLapTime.c

```diff
@@ -28,12 +29,25 @@
 
 EventEnum DisplayLapTime_StopAfterLap(SoftTimer* lapTimer)
 {
+    /* Stop the lapTime timer */
+    SoftTimer_Stop(lapTimer);
 
+    /* Stops the power supply to the DriveMotors */
+    DriveHandler_StopDriving();
+
+    /* Emit a short beep */
+    Buzzer_beep(BUZZER_NOTIFY);
 }
 
-void DisplayLapTime_DisplayLapTime(void)
+void DisplayLapTime_DisplayLapTime(SoftTimer* lapTimer)
 {
-  
+    /* Shows the completed lap time on the OledDisplay */
+    #define TIME_LEN_MAX (10)
+    UInt16 time = SoftTimer_get(lapTimer);
+    Display_gotoxy(0, 2);
+    char strTime[TIME_LEN_MAX] = {0};
+    sprintf(strTime, "%.2u:%0.3u", time / 1000U, time % 1000U);
+    Display_write(strTime, TIME_LEN_MAX);
 }
 
```

Comment:
SoftTimer counts down, SoftTimer_get returns time left not time passed

Reply:
Isnt that exactly what the countdown should do, display the time that is left?